lib/nrf_modem_lib/nrf_modem_lib.c

```diff
@@ -69,6 +71,27 @@ static const struct nrf_modem_bootloader_init_params bootloader_init_params = {
 	.shmem.size = PM_NRF_MODEM_LIB_SRAM_SIZE,
 	.fault_handler = nrf_modem_fault_handler
 };
+#endif /* CONFIG_SOC_SERIES_NRF91X */
+
+#ifdef CONFIG_SOC_SERIES_NRF92X
+
+static const struct nrf_modem_init_params init_params = {
+	.shmem.ctrl = {
+		.base = DT_REG_ADDR(DT_NODELABEL(cpuapp_cpucell_ipc_shm)),
+		.size = CONFIG_NRF_MODEM_LIB_SHMEM_CTRL_SIZE,
```

Comment:
Could we use DT for size as well and get rid of the KConfig values for nRF92?

Reply:
No, not currently. It must be set to the value of `CONFIG_NRF_MODEM_SHMEM_CTRL_SIZE` exported by `nrf_modem`. This is not in the devicetree. Let's see if we can improve this in some later work.